config/routes.rb

```diff
@@ -2,5 +2,5 @@
   mount RailsAdmin::Engine => '/admin', as: 'rails_admin'
   devise_for :users
   devise_for :user_admins
-
+  resources :users, only: [:index, :show, :create, :update, :destroy]
```

Comment:
remove the constraint. Let's allow the `resources` method provide you with all the routes you need.